File: joseffe-barroso/Python/aula04/while/exercise37.py
value = 1

while (value < 20):
    for index in range(1, 21, 1):
        result = value * index 
        print(f"{value} X {index} = {result}")
        index = index + 1

        
        if (index == 21):
            confirmation = input("Digite 'OK' para continuar: ")
            
            if (confirmation == 'OK'):
                value = value + 1
            else:
                confirmation = input("Digite 'OK' para continuar: ")

print("Thank you")

File: joseffe-barroso/Java/src/aula04/While/Exercise37.java
package While;

import java.util.Scanner;

public class Exercise37 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		String confirmation;
		int value, index, result;
		
		value = 1;
		
		while (value < 20) {
			for (index = 1; index < 21; index++) {
			    result = value * index;
			    System.out.printf("\n%d X %d = %d", value, index, result);
			    
			    if (index == 20) {
			    	System.out.print("\nDigite 'OK' para continuar: ");
					confirmation = scanner.next();
					
					while (!confirmation.equals("OK")) {
						System.out.print("\nDigite 'OK' para continuar: ");
						confirmation = scanner.next();
					}
					
					value = value + 1;
			    }
			}
		}
		
		System.out.print("Thank you");
	}

}
